# BI
###### Bioinformatics
- Corso del 2° Anno di Magistrale (1° Semestre).
- Docente: **Monica Bianchini**.
- [Teaching](https://www3.diism.unisi.it/~monica/bioinfo.html)
- [Kurzagast Video (Youtube)](https://www.youtube.com/watch?v=TYPFenJQciw&t=27s)
---
### Final Project
- [[BI - Project - ]]

---
## Perquisites:
- [[BI - Biology Introduction]]
- [[BI - Ripasso di Chimica e Biologia]]
---
### Concepts
- Lecture 1- Molecular Biology and Biochemistry
	- [[BI - Introduction to Molecular Biology and Biochemistry]]
	- [[BI - How big are the DNA data]]
	- [[BI - The Genetic Material]]
	- [[BI - The central Dogma of Molecular Biology]]
	- [[BI - Gene Structure]]
	- [[BI - The Genetic Code]]
	- [[BI - Introns and Exons]]
	- [[BI - The Protein Function]]
	- [[BI - The Primary Structure]]
	- [[BI - Tertiary and Quaternary Structure]]
	- [[BI - The Nature of Chemical Bonds]]
	- [[BI - Atom's Anatomy]]
	- [[BI - Valence]]
	- [[BI - Electronegativity]]
	- [[BI - Hydrogen Bonds]]
	- [[BI - Hydrophilic & Hydrophobic]]
	- [[BI - Molecular Biology Tools]]
	- [[BI - Restriction Enzymes]]
	- [[BI - Gel Electrophoresis]]
	- [[BI - Blotting and Hybridization]]
	- [[BI - Micorarrays]]
	- [[BI - Cloning]]
	- [[BI - Polymerase Chain Reaction (PCR)]]
	- [[BI - DNA Sequencing]]
	- [[BI - The C Value Paradox]]
	- [[BI - Reassociation Kinetics]]
- Lecture 2 - Database Search and Pairwise Alignments
	- [[BI - Introduction to Database Search and Pairwise Alignments]]
	- [[BI - Dot Plots]]
	- [[BI - Simple Alignments]]
	- [[BI - Penalities for the Presence of Gaps]]
	- [[BI - Score Matricies]]
	- [[BI - PAM Matricies]]
	- [[BI - BLOSUM Matricies]]
	- [[BI - PAM or BLOSUM]]
	- [[BI - The Needleman-Wunsch Algorithm]]
	- [[BI - Global and Local Alignments]]
	- [[BI - The Smith-Waterman Algortihm]]
	- [[BI - Biological Data and Databases]]
	- [[BI - Primary Biobanks]]
	- [[BI - NCBI]]
	- [[BI - Protein Databanks]]
	- [[BI - Specialized Databases]]
	- [[BI - Database Search]]
	- [[BI - BLAST and its Variants]]
	- [[BI - FASTA and its Variants]]
	- [[BI - Alignment Scores]]
	- [[BI - Multiple Alignments]]
- Lecture 3 - Substitution Patterns
	- [[BI - Introduction to Substitution Patterns]]
	- [[BI - Genes and Proteins]]
	- [[BI - How Proteins can Change]]
	- [[BI - Substitution Patterns in Genes]]
	- [[BI - Mutation Frequencies]]
	- [[BI - Functional Constraints]]
	- [[BI - Synonimous and Non-Synonymous Substitutions]]
	- [[BI - Indel and Pseudogenes]]
	- [[BI - Mutations and Substitutions]]
	- [[BI - Genetic Drift and Fixation]]
	- [[BI - Estimation of the Substitution Number]]
	- [[BI - Multiparametric Models]]
	- [[BI - Substitutions in Protein Sequences]]
	- [[BI - Evolutionary Speed Variations]]
	- [[BI - Molecular Clocks]]
- Lecture 4 - Distance based Phylogenetic Methods
	- [[BI - Homeoplasy and Homology]]
	- [[BI - Introduction to Distance-based Phylogenetic Methods]]
	- [[BI - Advantages of Molecular Phylogenies]]
	- [[BI - Phylogenetic Trees]]
	- [[BI - Trees of Genes versus Trees of Species]]
	- [[BI - Character and Distance Data]]
	- [[BI - Hierarchical Clustering]]
	- [[BI - Distance Matrix-based Methods]]
	- [[BI - Arc Length Estimation]]
	- [[BI - Transformed Distance Method]]
	- [[BI - Proximity Relation Methods]] #TODO 
	- [[BI - Neighbor-Joining Methods]] #TODO 
	- [[BI - Multiple Alignments]]
- Lecture 5 - Character-based Phylogenetic Methods
	- [[BI - Introduction to Character-based Phylogenetic Methods]]
	- [[BI - Informative and Non-Informative Sites]]
	- [[BI - Unweighted Parsimony]]
	- [[BI - Weighted Parsimony]]
	- [[BI - Quick Search Strategies ‘Branch and Bound’]]
	- [[BI - Quick Search Strategies ‘Heuristic Search’]]
	- [[BI - Quick Search Strategies ‘Consensus Trees’]]
	- [[BI - Tree Confidence 'Bootstrapping' and 'Parametric Test']]
	- [[BI - Molecular Phylogenies]]
	- [[BI - The Tree of Life]]
	- [[BI - The Origin of Man]]
- Lecture 6 - Neural Networks for Structured Data
	- [[BI - Introduction to Neural Networks for Structured Data]]
	- [[BI - Static and Dynamic Networks]]
	- [[BI - Elman Networks]]
	- [[BI - Jordan Networks]]
	- [[BI - Recurrent Networks]]
	- [[BI - Backpropagation Through Time]]
	- [[BI - Dynamics of a Neuron with Feedback]]
	- [[BI - The Vanishing or Exploding Gradient Problem]]
	- [[BI - LSTM ‘Long-Short Term Memories’]]
	- [[BI - Recursive Neural Networks]]
	- [[BI - Collisions]]
	- [[BI - Linear Recursive Networks]]
	- [[BI - Recursive Models for Non-Positional Graphs]]
	- [[BI - Recursive Models for Cycling Graphs]]
- Lecture 7 - Convolutional Neural Networks
	- [[BI - CNN ‘Convolutional Neural Networks’]]
	- [[BI - How to Make Up for the Lack of Data]]
	- [[BI - Deep Network Design]]
- Lecture 8 - Unsupervised Models and Clusterin
	- [[BI - Introduction to Unsupervised Learning]]
	- [[BI - Self Organizing Networks]]
	- [[BI - Self Organizing Maps]]
- Lecture 9 - Genomics and Gene Recognition
	- [[BI - Introduction to Genomics and Gene Recognition]]
	- [[BI - Genomics]]
	- [[BI - The Genome of Prokaryotes]]
	- [[BI - Structure of Prokaryotic Genes]]
	- [[BI - Promoter Elements]] #TODO 
	- [[BI - What is an Operon]]
	- [[BI - What are Ribosomes]]
	- [[BI - Open Reading Frames]]
	- [[BI - Conceptual Translation]]
	- [[BI - Terminator Sequences]] #TODO 
	- [[BI - GC Content in Prokaryotic Genomes]]
	- [[BI - Horizontal Gene Transfer]]
	- [[BI - Prokaryotic Gene Density]]
	- [[BI - The Genome of Eukaryotes]]
	- [[BI - Eukaryotic Gene Structure]]
	- [[BI - Promoter Elements]]
	- [[BI - Binding Sites of Regulatory Proteins]] #TODO 
	- [[BI - Open Reading Frames]]
	- [[BI - Introns and Exons]]
	- [[BI - Alternative Splicing]]
	- [[BI - GC Content in Eukaryotic Genomes]]
	- [[BI - CpG Islands]]
	- [[BI - Isochores]]
	- [[BI - Preferences in the use of Codons]]
	- [[BI - Gene Expression]]
	- [[BI - cDNA and ESTs]]
	- [[BI - Serial Analysis of Gene Expression]]
	- [[BI - Micorarrays]]
	- [[BI - Pharmacogenomics]]
	- [[BI - Transposition]]
	- [[BI - Repeated Elements]]
	- [[BI - Eukaryotic Gene Density]]
- Lecture 10 - Prediction of Protein and RNA Structure
	- [[BI - pH]]
	- [[BI - Introduction to Prediction of Protein and RNA Structure]]
	- [[BI - Amino Acids]]
	- [[BI - Backbone Flexibility (Protein Secondary Structure)]]
	- [[BI - Prediction Accuracy (Protein Secondary Structure)]]
	- [[BI - Chou-Fasman Method (Protein Secondary Structure)]]
	- [[BI - The GOR Method (Protein Secondary Structure)]]
	- [[BI - Neural Network Method (Protein Secondary Structure)]]
	- [[BI - Tertiary and Quaternary Structure]]
	- [[BI - Hydrophobicity (Tertiary and Quaternary Structure)]]
	- [[BI - Disulfide Bonds (Tertiary and Quaternary Structure)]]
	- [[BI - Active and Stable Structures (Tertiary and Quaternary Structure)]]
	- [[BI -  Algorithms for Protein Folding Modeling]]
	- [[BI - Lattice Models]] #TODO protein embedding
	- [[BI - Off-Lattice Models]]
	- [[BI - Folding Algorithms]]
	- [[BI - Misfolding]]
	- [[BI - Tertiary Structure Prediction]]
	- [[BI - Comparative Modeling]]
	- [[BI - Threading]]
	- [[BI - RNA Secondary Structure Prediction]] #TODO What are Ribozymes?
- Lecture 11 - Proteomics: Tools and Appliction
	- [[BI - Introduction to Proteomics - Tools and Appliction]]
	- [[BI - From Genome to Proteome]]
	- [[BI - Enzymes - An Introduction]]
	- [[BI - Enzymes]]
	- [[BI - Environmental Factors for Enzymes]]
	- [[BI - Enzyme Nomenclature]]
	- [[BI - Families and Superfamilies]]
	- [[BI - Folds]]
	- [[BI - Experimental Techniques]]
	- [[BI - Experimental Techniques ‘Two-Dimensional Electrophoresis’]]
	- [[BI - Experimental Techniques ‘Mass Spectromery’]]
	- [[BI - Experimental Techniques ‘Protein Microarrays’]]
	- [[BI - Inhibitors and Drug Design]]
	- [[BI - Drug Design]]
	- [[BI - Ligand Screening]]
	- [[BI - Ligand Docking]]
	- [[BI - Database Screening]]
	- [[BI - X-Ray Crystallography]]
	- [[BI - NMR Structures]]
	- [[BI - X-Ray Crystallography vs NRM]]
	- [[BI - Electron Microscopy Structures]]
	- [[BI - PDB (Protein Data Bank)]]
	- [[BI - Protein-Protein Interaction]]
	- [[BI - Protein Sorting]]
	- [[BI - Proteolytic Cleavage]]
	- [[BI - Glycosylation]]
	- [[BI - Phosphorylation]]
	- Use this to set ChatGPT: ***You are an Expert of Bioinformatics, please answer my questions***

----
# Original Slides
- [[BI - Lecture 0 - Introduction to the Course]]
- [[BI - Lecture 1 - Molecular Biology and Biochemistry]]
- [[BI - Lecture 2 - Database Search and Pairwise Alignments]]
- [[BI - Lecture 3 - Substitution Patterns]]
- [[BI - Lecture 4 - Distance Based Phylogentic methods]]
- [[BI - Lecture 5 -]] #TODO 
- [[BI - Lecture 6 -]] #TODO 
- [[BI - Lecture 7 - Convolutional Neural Networks]]
- [[BI - Lecture 8 - Unsupervised Modes and Clustering]]
- [[BI - Lecture 9 - Genomics and Gene Recognition]]
- [[BI - Lecture 10 - Prediction of Protein and RNA Structure]]
- [[BI - Lecture 11 - Proteomics Tools and Applications]]

---
# Questions
- ***Why do we use just the final part of the cDNA for ESTs (Expressed Sequence Tags)?***
	- Because the final part before the (**AAA**...**A** sequence) is **usually** unique (it's also short so it's more fast to search for it using algorithms)
- ***Percentage of active genes in Prokaryotes and Humans***:
	- $85\%$-$88\%$ of the **Prokaryotes** DNA are **active genes**, **coding DNA** (the "DNA spam" is only $25\%$ at most), in the Human DNA the **coding DNA** is just $3\%$.
- ***Cos'è il "trasfermento orizzontale dei geni" negli eucarioti?***
	- Si tratta di una mutazione del DNA di una creatura, per esempio mutato da un virus ad RNA, il trasferimento orizzontale è molto utile soprattutto per le creature che si riproducono in modo asesuato (il loro DNA non muta molto tra diverse generazioni)
- ***Why does the DNA length keeps increasing over new generations?***
	- Con l'andare del tempo il DNA di un organismo è destinato ad allungarsi, principalemente colpa dei **trasposoni**, specialmente il DNA degli eucarioti, anche se ci sono dei fenomeni di taglio (soprattutto in sequence ripetute, il DNA polimerasi "perde il filo"), ma in ogni caso in generale ==la lunghezza del DNA tenderà ad aumentare==.
- ***One DNA amminoacid sequnce does not produce always the same mRNA***: #NOT_SURE_ABOUT_THIS 
	- Once the DNA is traduced in RNA, the RNA is then modified if needed, for example the **intrones** are removed, what we are left with is the mRNA that will be traduced into a protein.
- ***Does the same mRNA produce the same protein in every species?*** #NOT_SURE_ABOUT_THIS
	- In the eukaryotes the same mRNA does produce the same protein for every individual.
- ***What is the purpose of the Lattice Model?*** #NOT_SURE_ABOUT_THIS 
	- It's just to model the central carbon $C_{\alpha}$, it does not find anything else that is actually useful
- ***What do tumors originate from?***
	- Form a different transcription and/or post-transcription of proteins, we can see this from proteomics studies, and studying the RNA of the cancer cell, this is the most common type of cancer
	- From a mutation of the DNA, a cancer devoleped this way is way more rare than the previous case
- ***What does an Enzyme do?***
	- Enymes are **Catalyst** (speed up processes)
		- Enzymes form complex tissues (like blood, blood is a complex tissue)
		- Enxymes also breake up big proteins, into smaller ones know as **products**, which are more easly menaged.
	- Enzymes are **not** consumed once used
	- Enzymes are extreamely specific (they accept only one type of protein)
	- ==Proteins change their shape during their life==, the enzymes need a particular **active site** to bind an work on their protein.
- ***How are Cofactors and Coenzimes attached and deatached to an Enzyme?***
	- A protein binds a Cofactor or Coenzyme to an Enzyme.
	- And another protein separeates them, when it is not needed anymore.
- ***?***
- ***?***
###
---
# Questions
###### How Big is the Human DNA?
- Total size of human genome: each cell carries 3.2 billion base pairs

###### What is the goal of Bioinformatics?
1. To understand the basis of biological diversity and to trace the evolutionary history of the life on the Earth, which is written in our molecules
2. To explain normal biological processes, to highlight malfunctions which lead to diseases, and to define approaches that can improve drug discovery and design

###### What does DNA stands for?
- DeoxyriboNucleic Acid

###### What is DNA made of?
- DNA molecules are made up of a few kinds of atoms ⎯ **carbon**, **hydrogen**, **nitrogen**, **oxygen** and **phosphorus**
- DNA molecules use only four nucleo-bases, **guanine**, **adenine**, **thymine** and **cytosine** (**G**, **A**, **T**, **C**), which are attached to a **phosphate group** (PO$_{4}$ ) and to a **deoxyribose sugar** (C$_5$H$_{10}$O$_4$), to form a **nucleotide**
- The nucleotides are divided into 2 groups **Purines** and **Pyrimidines**, we make this distinction because in the double helix of the DNA Guanine binds only with Adenine (both Purines) and Thymine binds only with Cytosine (both Pyrimidines) 
![[Pasted image 20221018105709.png]]

###### What is the difference between nucleo-bases and nucleotides?
- ***Nucleo-Bases*** or ***Bases***: Adenine Cytosine Thymine and Guanine (**A**, **C**, **T**, **G**)
![[Pasted image 20221018110208.png]]
- ***Nucleotides***: the Bases attached to a **phosphate group** (PO$_{4}$ ) and to a **deoxyribose sugar** (C$_5$H$_{10}$O$_4$).

###### What is the “function” of DNA?
- It is actually the information contained in the DNA that allows the organization of inanimate molecules in living cells and organisms, capable of regulating their internal chemical composition and their growth and reproduction
- It is also the DNA that gives us the inheritance of our ancestors’ physical traits, through the transmission of **genes**
- **Genes** contain the information, in the form of specific nucleotide sequences, which constitute the DNA molecules

###### What is a Gene?
- A gene is **a sequence of nucleotides along a strand of DNA that a cell nucleus uses to produce proteins**. Genes determine the specific traits of an organism.
- Complicated genes may be composed by hundreds of nucleotides

###### What is the Genome?
- The genetic code “which describes” an organism, known as its genome, is conserved in millions/billions of nucleotides

###### What is a Chromosome?
- [[BI - Chromosomes]]

######
---
###### What is the Phenotype?
- The physical aspect of an individual (or animal, bacteria, …)

###### What are the Taxonomic Units?
###### What are the differences between Substitution and Indel Events?
###### QUESTION?
###### QUESTION?
###### QUESTION?
###### QUESTION?
###### QUESTION?
###### QUESTION?
###### QUESTION?
###### QUESTION?
---
# Project: \[Velocit\]
- [[BI - Paper 'Velocità di traduzione'.pdf]]
----
###### All My Notes
For the best experience in reading these and all other notes, and also if you wish to EDIT them, do as follows: 
1. Install [Obsidian](https://obsidian.md), or another markdown editor.
2. Go to the Github link of this or another note
3. Download all the repo or if you know git just the 'content/' folder
4. Extract just the 'content/' folder from the repo zip file
5. Open Obsidian >> Menage Vaults >> Open Folder as Vault >> and select the 'content/' folder you just extracted

==PLEASE NOTE==:
- These notes were not revised by the professors, so take all of them with a grain of salt.
- However if you download them since they are made in markdown you can EDIT them, please do so.
- If you edit and "upgrade" them, please pass the new ones to the other students and professors.

Here are all the links to my notes:
- ***Github***: [UNISI-Sensors-and-Microsystems-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Sensors-and-Microsystems-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Sensors-and-Microsystems-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Sensors-and-Microsystems-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-Complex-Dynamic-Systems-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Complex-Dynamic-Systems-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Complex-Dynamic-Systems-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Complex-Dynamic-Systems-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-Discrete-Event-Systems-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Discrete-Event-Systems-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Discrete-Event-Systems-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Discrete-Event-Systems-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-System-Identification-and-Data-Analysis-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-System-Identification-and-Data-Analysis-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-System-Identification-and-Data-Analysis-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-System-Identification-and-Data-Analysis-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-Multivariable-NonLinear-and-Robust-Control-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Multivariable-NonLinear-and-Robust-Control-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Multivariable-NonLinear-and-Robust-Control-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Multivariable-NonLinear-and-Robust-Control-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-Artificial-Intelligence-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Artificial-Intelligence-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Artificial-Intelligence-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Artificial-Intelligence-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-Human-Centered-Robotics-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Human-Centered-Robotics-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Human-Centered-Robotics-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Human-Centered-Robotics-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-Machine-Learning-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Machine-Learning-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Machine-Learning-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Machine-Learning-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-Bioinformatics-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Bioinformatics-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Bioinformatics-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Bioinformatics-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-Network-Optimization-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Network-Optimization-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Network-Optimization-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Network-Optimization-Obsidian-Quartz-Publish).
- ***Github***: [UNISI-Mathematical-Methods-for-Engineering-Obsidian-Quartz-Publish](https://github.com/Uomocosa/UNISI-Mathematical-Methods-for-Engineering-Obsidian-Quartz-Publish);<br>***Quartz Publish***: [UNISI-Mathematical-Methods-for-Engineering-Obsidian-Quartz-Publish](https://uomocosa.github.io/UNISI-Mathematical-Methods-for-Engineering-Obsidian-Quartz-Publish).
