##### Questions
- ***What are Microarrays?***
	- Microarrays, also known as DNA chips or gene chips, are powerful tools used in molecular biology and genetics research to measure the expression levels of thousands of genes simultaneously. <br>==Microarrays consist of a solid support, such as a glass slide or a silicon chip, that has thousands of microscopic spots, or probes, attached to its surface==.
	- ==Each probe on a microarray is designed to capture and hybridize with a specific mRNA molecule in a sample of cells or tissues==. <br>The intensity of the hybridization signal at each spot on the microarray is proportional to the abundance of the corresponding mRNA molecule in the sample, allowing researchers to measure the expression levels of thousands of genes in a single experiment.
	- Microarrays can be used to study gene expression patterns in many different contexts, including development, disease, and drug discovery. <br>They have also been used to identify genes that are differentially expressed between different cell types, tissues, or experimental conditions, and to study the effects of genetic mutations and environmental factors on gene expression.
	- In addition to their use in gene expression analysis, microarrays have also been used to study DNA sequence variation, epigenetic modifications, and protein-protein interactions. <br>Microarrays have revolutionized the field of genomics and have contributed significantly to our understanding of gene expression and regulation.

##### —————————————————————
##### IMPORTANTE

> #IMPORTANTE ***DNA Microarrays***
> ==A reverse hybridization technique to search a particular sequence, a particular DNA fragment, in a bunch of DNA fragments==.
> 
> A DNA microarrary is a silicon chip in which DNA probes are fixed when created, each probe correspond to a specific cDNA fragment, and during a reverse hybridization process.
> DNA fragments we want to study are marked with radioactive isotopes, and then mixed with a salty solution, at 100° C, finally we deposit this solution in the microarray.
> We study which fragment is present in the microarray based on which mark (the radioactive isotopes) are present and in which position.
> 
> ***NOTE***: Each dot (“pozzetto”) in the matrix is typically less than 200 microns in size and is made up of many copues of the same DNA sequence (or probe), we call this dot a **feature**, the basic unit of a microarray.
> 
> ***NOTE***: Usually the DNA fragments we study are taken from mRNA found in an organism, we take its mRNA, and convert it in cDNA, mark it and then pour it in the microarray.
> 
> ***NOTE***: We take two target sequence (~ex.: one diseased and one normal/sane) and mark them in different color, to see what DNA fragments have in common and which one are unique.
> 
> ***NOTE***: The hybridization condition: **salt concentration** and **temperature**, can be changed to allow the coupling of not perfect complementary sequences.
> *~Ex.: changing the hybridization conditions we can allow the two fragments **ATTCG** and **TATGC** to bind, even if they are not perfect complementary sequences*

> #IMPORTANTE ***cDNA Microarrays***:
> A cDNA microarray uses **ESTs** (one helix fragment taken from the cDNA library) as probes fixing them in small glass or silicon slides.
> Then we can mark the cDNA taken from the normal and diseased cell of organism we want to study, and mark them in different fluorescent colors.
> We put the marked cDNAs in the microarray and watch the resulting colors, to determine the gene expression of the normal tissue/cell and diseased one.
> ![[Pasted image 20230212202816.png]]
> ***Online Resource***: [Youtube](https://www.youtube.com/watch?v=JGknms7awlU) 

##### —————————————————————
##### Slides with Notes
![[Lecture 1-104.png]] ![[Lecture 1-105.png]]

> #IMPORTANTE ***DNA Microarrays***
> ==A reverse hybridization technique to search a particular sequence, a particular DNA fragment, in a bunch of DNA fragments==.
> 
> A DNA microarrary is a silicon chip in which DNA probes are fixed when created, each probe correspond to a specific cDNA fragment, and during a reverse hybridization process.
> DNA fragments we want to study are marked with radioactive isotopes, and then mixed with a salty solution, at 100° C, finally we deposit this solution in the microarray.
> We study which fragment is present in the microarray based on which mark (the radioactive isotopes) are present and in which position.
> 
> ***NOTE***: Each dot (“pozzetto”) in the matrix is typically less than 200 microns in size and is made up of many copues of the same DNA sequence (or probe), we call this dot a **feature**, the basic unit of a microarray.
> 
> ***NOTE***: Usually the DNA fragments we study are taken from mRNA found in an organism, we take its mRNA, and convert it in cDNA, mark it and then pour it in the microarray.
> 
> ***NOTE***: We take two target sequence (~ex.: one diseased and one normal/sane) and mark them in different color, to see what DNA fragments have in common and which one are unique.
> 
> ***NOTE***: The hybridization condition: **salt concentration** and **temperature**, can be changed to allow the coupling of not perfect complementary sequences.
> *~Ex.: changing the hybridization conditions we can allow the two fragments **ATTCG** and **TATGC** to bind, even if they are not perfect complementary sequences*

![[Lecture 1-106.png]] ![[Lecture 1-107.png]] ![[Lecture 1-108.png]] ![[Lecture 1-109.png]] 


![[BI - Lecture 9 - Genomics and gene recognition-143.png]] ![[BI - Lecture 9 - Genomics and gene recognition-144.png]] 
