##### Questions
- ***?***

---
##### IMPORTANTE

> #IMPORTANTE ***BLAST and its Variants***:
> **BLAST**: search for long local alignments without gaps
> **BLASTN**: Uses the following matrix to search for good scores between two sequences, still without gaps <br>![[Pasted image 20230319174437.png]]
> **BLASTX**: Like **BLASTN** but also translates the nucleoutide sequence into the protein sequence.
> **BLASTP**: Searches for the exact match of a sequence of usually 4 amino acids in the database, than creates a query adding other matching amino acids. *~Ex.:*<br>![[Pasted image 20230319174818.png]]
> **BLAST 2.0**: insert gaps
> **BLAST+**: command line tools, for using your own BLAST algorithm in local.
> **Magic-BLAST**: tool for mapping large next-generation RNA or DNA sequencing, it runs against a whole genome or transcriptome.

---
##### Slides with Notes
![[Lecture 2-097.png]] ![[Lecture 2-098.png]] ![[Lecture 2-099.png]] ![[Lecture 2-100.png]] ![[Lecture 2-101.png]]

> #IMPORTANTE ***BLAST and its Variants***:
> **BLAST**: search for long local alignments without gaps
> **BLASTN**: Uses the following matrix to search for good scores between two sequences, still without gaps <br>![[Pasted image 20230319174437.png]]
> **BLASTX**: Like **BLASTN** but also translates the nucleoutide sequence into the protein sequence.
> **BLASTP**: Searches for the exact match of a sequence of usually 4 amino acids in the database, than creates a query adding other matching amino acids. *~Ex.:*<br>![[Pasted image 20230319174818.png]]
> **BLAST 2.0**: insert gaps
> **BLAST+**: command line tools, for using your own BLAST algorithm in local.
> **Magic-BLAST**: tool for mapping large next-generation RNA or DNA sequencing, it runs against a whole genome or transcriptome.
