##### Questions
- ***What are Reassociation Kinetics?***
	- ==Reassociation kinetics is a method used to analyze the complexity and size of DNA or RNA samples==. <br>This method involves measuring the rate at which single-stranded nucleic acids anneal or reassociate to form double-stranded molecules under specific conditions.
	- ==In reassociation kinetics, denatured single-stranded DNA or RNA is allowed to reassociate under controlled conditions, such as temperature and salt concentration==. <br>==The rate of reassociation is then measured and used to infer the complexity and size of the original nucleic acid sample==.
	- The rate of reassociation is influenced by the concentration of the nucleic acid sample, as well as its length and degree of sequence complexity. <br>More complex nucleic acid samples (e.g. genomes) will take longer to reassociate than less complex samples (e.g. single genes), as there will be more possible combinations of complementary sequences to form double-stranded molecules.
	- Reassociation kinetics has been used to study a variety of biological questions, such as the genome size and organization of different species, the presence of repetitive elements in genomes, and the formation of hybridization between different DNA or RNA samples. <br>The method has also been used to estimate the complexity and size of nucleic acid samples in environmental samples, such as soil and water, where traditional sequencing methods may be difficult to implement.

---
##### IMPORTANTE

> #IMPORTANTE ***Reassociation Kinetics***
> 1. We denatour the DNA of an organism
> 2. We let the separeted DNA reform its preavious structure (reassociation)
> 3. We calculate the ***cot equation***: $$\frac{c_t}{c_0} = \frac{1}{1 + k \kern2px c_0 \kern2pxt}$$Where: 
>		- $c_t$ is the single-strand DNA concentration at time $t$
>		- $c_0$ is the initial concentration
>		- $k$ is a constant dependent on temperature, on the size of the fragment, and the complexity of the nucleotide sequence
>
> We calculate the time $t_{1/2}$, which means how much time it takes such that $\frac{c_t}{c_0} = \frac{1}{2}$.
> And we can think of the product $c_0 \kern2px t_{1/2}$ as a score, of the amount of “unique” information encoded in a genome.
> The C value (the lenght of the DNA of an organism) is not a value that describe the complexity of the organism, while $c_0 \kern2px t_{1/2}$ can be.
> Since a higher “cot score” means less **junk DNA**

---
##### Slides with Notes
![[Lecture 1-139.png]] ![[Lecture 1-140.png]]

> #IMPORTANTE ***Reassociation Kinetics***
> 1. We denatour the DNA of an organism
> 2. We let the separeted DNA reform its preavious structure (reassociation)
> 3. We calculate the ***cot equation***: $$\frac{c_t}{c_0} = \frac{1}{1 + k \kern2px c_0 \kern2pxt}$$Where: 
>		- $c_t$ is the single-strand DNA concentration at time $t$
>		- $c_0$ is the initial concentration
>		- $k$ is a constant dependent on temperature, on the size of the fragment, and the complexity of the nucleotide sequence
>
> We calculate the time $t_{1/2}$, which means how much time it takes such that $\frac{c_t}{c_0} = \frac{1}{2}$.
> And we can think of the product $c_0 \kern2px t_{1/2}$ as a score, of the amount of “unique” information encoded in a genome.
> The C value (the lenght of the DNA of an organism) is not a value that describe the complexity of the organism, while $c_0 \kern2px t_{1/2}$ can be.
> Since a higher “cot score” means less **junk DNA**

![[Lecture 1-141.png]] ![[Lecture 1-142.png]] ![[Lecture 1-143.png]] ![[Lecture 1-144.png]]
